app/views/external_users/claims/case_details/_cracked_trial_fields.html.haml

```diff
@@ -6,12 +6,14 @@
 
   = f.gov_uk_date_field(:trial_fixed_at, legend_text: t('.trial_fixed_at'), legend_class: 'govuk-legend', form_hint_text: t('.date_hint'), id: "trial_fixed_at",error_messages: gov_uk_date_field_error_messages(@error_presenter, :trial_fixed_at))
 
-  = f.gov_uk_date_field(:trial_cracked_at, legend_text: t('.trial_cracked_at'), legend_class: 'govuk-legend', form_hint_text: t('.date_hint'), id: "trial_cracked_at", error_messages: gov_uk_date_field_error_messages(@error_presenter, :trial_cracked_at))
+  - # = TODO: this needs testing
```

Comment:
Can this comment be removed? Is it covered by features/page_objects/sections/cracked_trial_section.rb ?